txscript/data/script_tests.json

```diff
@@ -1365,87 +1459,193 @@
 ["-11 11", "GREATERTHANOREQUAL NOT", "NONE", "OK"],
 ["-11 -10", "GREATERTHANOREQUAL NOT", "NONE", "OK"],
-["1 0 MIN", "0 NUMEQUAL", "NONE", "OK"],
+["1 2147483648", "GREATERTHANOREQUAL", "NONE", "ERR_OUT_OF_RANGE", "GREATERTHANOREQUAL must fail with stack item >4 bytes"],
+
+["MIN test coverage"],
+["NOP", "MIN", "NONE", "ERR_INVALID_STACK_OPERATION", "MIN requires two items on the stack and NOP must not be treated as one"],
+["", "MIN", "NONE", "ERR_INVALID_STACK_OPERATION", "MIN requires two items on the stack, stack has no items"],
+["0", "MIN", "NONE", "ERR_INVALID_STACK_OPERATION", "MIN requires two items on the stack, stack has one item"],
+["1 0 MIN", "0 NUMEQUAL", "NONE", "OK", "MIN must compare the first two numeric stack items and return the mimimum"],
 ["0 1 MIN", "0 NUMEQUAL", "NONE", "OK"],
 ["-1 0 MIN", "-1 NUMEQUAL", "NONE", "OK"],
 ["0 -2147483647 MIN", "-2147483647 NUMEQUAL", "NONE", "OK"],
-["2147483647 0 MAX", "2147483647 NUMEQUAL", "NONE", "OK"],
+["1 2147483648", "MIN", "NONE", "ERR_OUT_OF_RANGE", "MIN must fail with stack >4 bytes"],
+
+["MAX test coverage"],
+["NOP", "MAX", "NONE", "ERR_INVALID_STACK_OPERATION", "MAX requires two items on the stack and NOP must not be treated as one"],
+["", "MAX", "NONE", "ERR_INVALID_STACK_OPERATION", "MAX requires two items on the stack, stack has no items"],
+["0", "MAX", "NONE", "ERR_INVALID_STACK_OPERATION", "MAX requires two items on the stack, stack has one item"],
+["2147483647 0 MAX", "2147483647 NUMEQUAL", "NONE", "OK", "MAX must compare the first two numeric stack items and return the maximum"],
 ["0 100 MAX", "100 NUMEQUAL", "NONE", "OK"],
 ["-100 0 MAX", "0 NUMEQUAL", "NONE", "OK"],
 ["0 -2147483647 MAX", "0 NUMEQUAL", "NONE", "OK"],
-["0 0 1", "WITHIN", "NONE", "OK"],
+["1 2147483648", "MAX", "NONE", "ERR_OUT_OF_RANGE", "MAX must fail with stack item >4 bytes"],
+
+["WITHIN test coverage"],
+["NOP", "WITHIN", "NONE", "ERR_INVALID_STACK_OPERATION", "WITHIN requires three items on the stack and NOP must not be treated as one"],
+["", "WITHIN", "NONE", "ERR_INVALID_STACK_OPERATION", "WITHIN requires three items on the stack, stack has no items"],
+["0", "WITHIN", "NONE", "ERR_INVALID_STACK_OPERATION", "WITHIN requires three items on the stack, stack has one item"],
+["0 1", "WITHIN", "NONE", "ERR_INVALID_STACK_OPERATION", "WITHIN requires three items on the stack, stack has two items"],
+["0 0 1", "WITHIN", "NONE", "OK", "WITHIN must check if the third numeric stack item falls within the range of the second and first numeric stack items (left inclusive)"],
 ["1 0 1", "WITHIN NOT", "NONE", "OK"],
 ["0 -2147483647 2147483647", "WITHIN", "NONE", "OK"],
 ["-1 -100 100", "WITHIN", "NONE", "OK"],
 ["11 -100 100", "WITHIN", "NONE", "OK"],
 ["-2147483647 -100 100", "WITHIN NOT", "NONE", "OK"],
 ["2147483647 -100 100", "WITHIN NOT", "NONE", "OK"],
+["1 0 2147483648", "WITHIN", "NONE", "ERR_OUT_OF_RANGE", "WITHIN must fail with stack item >4 bytes"],
```

Comment:
Same here regarding needing another test for the other operands being >4 bytes and comment.  Note that there are 3 here instead of 2 as all of the others up to this point.